### ISSUE TITLE
BattleScribe Android app via F-Droid possible?

### ISSUE THREAD

**Issue by r4dh4l** [BattleScribe/Release#31]
Hi,

@amis92 [told me](https://github.com/BSData/bsdata/issues/216#issuecomment-466688814) to request this here: Would it be possible to provide the Android app via https://f-droid.org/ as well?

**Comment by r4dh4l** [comment]
...anyone? :)

**Comment by OftKilted** [comment]
YMMV ... but Battlescribe doesn’t appear to meet the core repository inclusion requirements https://f-droid.org/en/docs/Inclusion_Policy/ to be in the official f-droid.org repository.

**Comment by r4dh4l** [comment]
So which ones are the "show stopper"?

**Comment by amis92** [comment]
All of them, really. It's a proprietary software using ads via proprietary libraries, integrated with Play services, source code is not public, it's built with non floss tools...

A good wuestion would rather be, which requirements does BattleScribe actually meet. 

TLDR: no chance

**Comment by r4dh4l** [comment]
Oh, for some reason I didn't expected that. No way to make it FLOSS?

**Comment by amis92** [comment]
BattleScribe is adware or paid. How can it be [**F**OSS](https://en.wikipedia.org/wiki/Free_and_open-source_software)? You know, there is a license you accepted before using BattleScribe. ;)

**Comment by r4dh4l** [comment]
That's a common misunderstanding: The "free" in Free/(Libre/)Open Source Software is not the same "free" as in "free beer" but as in "free speech" or "freedom". Yes, a better term would be "freedom respecting software" and that's why the [OSI](https://en.wikipedia.org/wiki/Open_Source_Initiative) invented the term "open source software" many years _after_ the [FSF](https://en.wikipedia.org/wiki/Free_Software_Foundation), the "re-inventor" of Free Software, was founded. "Open Source" as term solved the problem not to mix "Free Software" with "[Freeware](https://en.wikipedia.org/wiki/Freeware)" but caused another problem: There is software which you can call "open", because you can take a look into the code, but the license does not provide you the original [4 freedoms](https://en.wikipedia.org/wiki/The_Free_Software_Definition#The_definition_and_the_Four_Freedoms) of Free Software so the more common term "open source" does not solve the problem in the end. Terms like "FOSS" or "FLOSS" want to combine both terms to express all aspects (but again: none of these aspects has something to do with "at no charge" as in "freeware").

Delevoping Free Software does not and did never mean to code something at no charge but to provide users the freedom to _use_, to _share_, to _understand_ and to _modify_ software. Best example for very successfull commercial FLOSS is Android because every smartphone company can take the [code](https://en.wikipedia.org/wiki/Android_(operating_system)#AOSP) and adjust it however they want or communities like [LineageOS](https://en.wikipedia.org/wiki/LineageOS) can develop Android versions without the common bloatware. All these coders needs to be payed in the same way as coders of proprietary software needs to be but thanks to FLOSS the development can be done much more sustainable (best example here: the [fork from OpenOffice to LibreOffice](https://en.wikipedia.org/wiki/LibreOffice#ooo-build,_Go-oo_and_Oracle) since Oracle bought Sun Microsystems).

However: Somehow I overlooked the license, especially because GitHub is in the most cases a place for developing FLOSS. My fault.

**Comment by amis92** [comment]
Of course, I understand the difference. However, BattleScribe is developed by a single person (@Jonskichov) and I'd be surprised if he shared/opened the source code, no matter the actual license.

**Comment by r4dh4l** [comment]
I'm surprised that a program for tabletop games which relies on community support a lot (providing and reviewing the unit data) is *not* developed as FLOSS. I mean: What does the community using BattleScribe does if @Jonskichov just "disappears" one day (no mood/no time to continue/illness/dead - so the general things that happen to human being)...

If it works this way after the last years and the community is fine with the situation I'm shouldn't be the one to judge it in any way. It's just my personal doubt that BattleScribe can survive as proprietary "one man show" because of the mentioned very natural reasons.
